lib/lhm/throttler/replica_lag.rb

```diff
@@ -11,7 +11,7 @@ def self.format_hosts(hosts)
       formatted_hosts
     end
 
-    class SlaveLag
```

Comment:
maybe we should have a separate second class that inherits from this one, or the other way around for the "deprecation" period, in case someone uses this class directly, wdyt?

Reply:
Good idea.

Just re-stated the `SlaveLag` throttler class, which inherits from the `ReplicaLag` throttler class.